release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -405,6 +405,16 @@ For more information, see xref:../authentication/managing_cloud_provider_credent
 
 {product-title} 4.9 introduces the following notable technical changes.
 
+[discrete]
+[id="octavia-ovn-nodeport-changes"]
+==== Octavia OVN NodePort changes
+Previously, opening traffic on NodePorts were constrained to the CIDR of the node's subnet. In order to support LoadBalancer services using the Octavia Open Virtual Network (OVN) provider, the security group rules that allow NodePort traffic to master and worker nodes are now changed to open `0.0.0.0/0`. 
```

Comment:
This definitely needs a notion that it's only affecting OpenStack deployments, otherwise it'll be scary for users running on other clouds.